src/data/disks.rs

```diff
@@ -145,8 +163,47 @@ extern "C" {
 
 #[cfg(not(windows))]
 fn load_disks(canonicalize_paths: bool) -> Vec<Disk> {
-    sysinfo::Disks::new_with_refreshed_list()
-        .iter()
-        .map(|d| Disk::from_sysinfo_disk(d, canonicalize_paths))
-        .collect()
+
+    let mut result: Vec<Disk> = Vec::new();
+    let mut seen_mount_points = std::collections::HashSet::new();
+
+    // Use sysinfo to get system disks
+    let disks = sysinfo::Disks::new_with_refreshed_list();
+    for disk in &disks {
+        let mount_point = disk.mount_point();
+        if mount_point == Path::new("/") {
+            // Skip the root system drive
+            continue;
+        }
+
+        // Ensure no duplicates by checking the mount point
+        if !seen_mount_points.insert(mount_point.to_path_buf()) {
+            continue;
+        }
+
+        result.push(Disk::from_sysinfo_disk(disk, canonicalize_paths));
+    }
+
+    // On macOS, add volumes from `/Volumes`
+    #[cfg(target_os = "macos")]
+    {
+        if let Ok(entries) = fs::read_dir("/Volumes") {
+            for entry in entries.filter_map(std::result::Result::ok) {
+                let path = entry.path();
+
+                // Skip already seen mount points
+                if !seen_mount_points.insert(path.clone()) {
+                    continue;
+                }
+
+                // Include only directories and ensure they are valid volumes
+                if let Ok(metadata) = entry.metadata() {
+                    if metadata.is_dir() {
+                        result.push(Disk::from_path(&path, canonicalize_paths));
+                    }
+                }
+            }
+        }
+    }
+    result
```

Comment:
Could you move this to a `load_disks` method for Mac OS only and leave the method unchanged for Linux and other operating systems?